.github/scripts/docker-test.sh

```diff
@@ -33,7 +33,7 @@ wait_for() {
   printf "OK\n"
 }
 
-apt-get update && apt-get upgrade -y && apt get install -y netcat
+sudo apt-get update && sudo apt-get upgrade -y && sudo apt-get install -y netcat
```

Comment:
```suggestion
```
https://github.com/netdata/netdata/runs/5514863217?check_suite_focus=true

> 101 upgraded, 0 newly installed, 0 to remove and 8 not upgraded.
54 standard security updates
> Need to get 535 MB of archives.
> After this operation, 21.9 MB of additional disk space will be used.

> **netcat is already the newest version (1.206-1ubuntu1).**

Reply:
We should not be relying on `netcat` being installed here (in the past it has not been by default, and I would not bet on it continuing to be the default), and if we0re installing packages we need to pull in any possible updates for consistency reasons (`APT can behave rather strangely if you try to install packages when you have pending updates).

Reply:
Done.